src/error.rs

```diff
@@ -18,7 +18,7 @@ use std::error::Error;
 #[derive(Debug)]
 pub enum RequestError {
     /// An HTTP communication error occurred while sending the request or receiving the response.
-    HyperError(HyperError),
+    HttpError(ReqwestError),
```

Comment:
Maybe you'd prefer naming this `ReqwestError`?

Reply:
Nah, `HttpError` is fine